### PR TITLE
display exit code

### PR DESCRIPTION
**Pull request by dickeyxxx**
also changed exit failure output to red

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by rmg** [comment]
@slnode ok to test

File: lib/console.js [hunk 1]
```diff
@@ -65,6 +65,11 @@ function Console(logger) {
     logger.log(proc.color(this.pad(stamp,this.padding)), colors.bright_cyan(string));
   };
 
+  this.error = function info(key, proc, string) {
```
Comment (rmg) [inline lib/console.js]: Should probably just drop the `info` name, or call it `error`.

File: lib/console.js [hunk 2]
```diff
@@ -65,6 +65,11 @@ function Console(logger) {
     logger.log(proc.color(this.pad(stamp,this.padding)), colors.bright_cyan(string));
```
Comment (rmg) [inline lib/console.js]: This `.Info()` method is no longer used, as far as I can tell. Probably a good time to delete it.

Comment (dickeyxxx) [inline lib/console.js]: updated

**Pull request merged**